components/style/gecko_string_cache/namespace.rs

```diff
@@ -25,17 +25,9 @@ macro_rules! ns {
 
 /// A Gecko namespace is just a wrapped atom.
 #[derive(
-    Clone,
```

Comment:
I don't think we need to add feature checks to any of the style/gecko/ or style/gecko_string_cache files.

Reply:
Good point. Amended.